test/testExplorer/TestExplorer.ts

```diff
@@ -55,7 +78,7 @@ const initTestExplorer = async (): Promise<void> => {
       subscriptionId: portalRunnerSubscripton,
       resourceGroup: portalRunnerResourceGroup,
       authorizationToken: `Bearer ${token}`,
-      features: {},
+      features: { sampleFeature: "sampleFeatureValue" },
```

Comment:
>sampleFeature [](start = 18, length = 13)

For now it is ok. When you fix so that index policy edito is shown in standalone with AAD, please come and fix it here.